### ISSUE TITLE
Reformat remarks editor

### ISSUE THREAD

**Issue by kjmahalingam** [synthetichealth/caremaps-react#68]
Since remarks are separated in code onto separate lines, they are also separated onto separate lines in the editor. They should be formatted as a continuous paragraph to remove the awkward line breaks.

**Comment by arscan** [comment]
We had some conversation in #224 regarding what this would entail.  See the following comment and the 3 comments after:

https://github.com/synthetichealth/module-builder/pull/224#issuecomment-411816970

**Comment by KeeyanGhoreshi** [comment]
There should really be a separate code representation of a line break, like an empty string in the array.   Or even better, the remarks should be stored as an array of arrays, with each array being one continuous section with no breaks at the cost of some readability.

**Comment by arscan** [comment]
I like empty string representing a forced line break.  An array of arrays would be a pain to write by hand and harder to explain.

The other major categories I know of are:
* Major visual breaks, like `================`
* links
* lists
* tables

Basically a subset of markdown. I looked into importing a javascript markdown renderer, and there are some, but didn't take the time to actually do it.  Seems a bit overkill, and this is a `low priority` issue.

**Comment by arscan** [comment]
Personally I think that we should be putting a LOT more information into these remarks if we want any hope of future people to edit a module once it has been created.  They all need to reference where the numbers come from, assumptions made, etc.

**Comment by arscan** [comment]
Yes, we would take a pass and manually reformat all existing modules.  We have done that before.
